expression/builtin_time_vec.go

```diff
@@ -697,11 +697,33 @@ func (b *builtinGetFormatSig) getFormat(format, location string) string {
 }
 
 func (b *builtinLastDaySig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinLastDaySig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	if err := b.args[0].VecEvalTime(b.ctx, input, result); err != nil {
+		return err
+	}
+	times := result.Times()
+	for i := 0; i < n; i++ {
+		if result.IsNull(i) {
+			continue
+		}
+		if times[i].InvalidZero() {
+			return handleInvalidTimeError(b.ctx,
+				types.ErrIncorrectDatetimeValue.GenWithStackByArgs(times[i].String()))
```

Comment:
If this error is handled here, we should continue to calculate the next row:
```
If err := handleInvalidTimeError(...); err != nil {
    return err
}
result.SetNull(i, true)
continue
```

Reply:
Do we need this judgment?
I do not see it in builtinLastDaySig.evalTime

Reply:
Fixed